bmeg_app/components/func_repurpose_drug.py

```diff
@@ -0,0 +1,177 @@
+from ..db import G
+import gripql 
+import pandas as pd 
+from collections import Counter
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+def get_matrix(PROJECT, DRUGRESPONSE):
```

Comment:
consistent variable format snakecase e.g.  s/DRUGRESPONSE/drug_response/

Reply:
Fixed in commit 
56d6a39